server/app/com/xsn/explorer/data/anorm/dao/BlockPostgresDAO.scala

```diff
@@ -307,6 +308,116 @@ class BlockPostgresDAO @Inject()(
     }
   }
 
+  def getBlocks(limit: Limit, orderingCondition: OrderingCondition)(implicit conn: Connection): List[BlockHeader] = {
```

Comment:
Then, you can create a new model specific for this query, and avoid updating the previous queries, it's cleaner

Reply:
Created `BlockInfo` model for this.